src/vue/components/panel-compact.vue

```diff
@@ -377,44 +405,16 @@ export default {
     },
 
     attachToLeftVisible: function () {
-      return (this.data && this.data.settings) ? this.data.settings.panelPosition.currentValue === 'right' : false
+      return this.panelPosition === 'right'
     },
 
     attachToRightVisible: function () {
-      return (this.data && this.data.settings) ? this.data.settings.panelPosition.currentValue === 'left' : true
+      return this.panelPosition === 'left'
     },
 
     // Need this to watch when inflections tab becomes active and adjust panel width to fully fit an inflection table in
     inflectionsTabVisible: function () {
-      // Inform an inflection component about its visibility state change
-      if (this.data && this.data.inflectionComponentData.inflectionViewSet) {
-        this.data.inflectionComponentData.visible = this.data.tabs.inflections
-      }
-      return this.data.tabs.inflections
-    },
-
-    // Need this to watch when inflections browser tab becomes active and adjust panel width to fully fit an inflection table in
-    inflectionsBrowserTabVisible: function () {
-      // Inform an inflection browser component about its visibility state change
-      if (this.data && this.data.inflectionBrowserData) {
-        this.data.inflectionBrowserData.visible = this.data.tabs.inflectionsbrowser
-      }
-      return this.data.tabs.inflectionsbrowser
-    },
-
-    treebankTabAvailable: function () {
-      // treebank data is possible if we have it for the word or the page
-      return !!(this.data && this.data.treebankComponentData && this.data.treebankComponentData.data &&
-          ((this.data.treebankComponentData.data.page && this.data.treebankComponentData.data.page.src) ||
-            (this.data.treebankComponentData.data.word && this.data.treebankComponentData.data.word.src)))
-    },
-
-    treebankTabVisible: function () {
-      // Inform treebank component about visibility state change
-      if (this.data && this.data.treebankComponentData && this.data.treebankComponentData.data) {
-        this.data.treebankComponentData.visible = this.data.tabs.treebank
-      }
-      return this.data.tabs.treebank
+      return Boolean(this.$store.state.app.tabState.inflections && this.$store.state.app.inflectionsViewSet)
```

Comment:
is this sufficient to handle resizing when the tables change?

Reply:
That worked well in my tests, but I might be missing some cases 🙂. I do think we need some clearer logic to handle resizing. It's too convoluted now.

Reply:
ok, it's okay to leave this as is for now and I will enter an issue to make sure we don't forget to address panel resizing across the board.

Reply:
Thanks!